BinaryConverter/src/BinaryConverter.java

```diff
@@ -0,0 +1,366 @@
+
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Scanner;
+
+import javax.swing.JOptionPane;
+
+public class BinaryConverter {
+	
+	public static final int FINAL_IEEE_BITS = 23;
+	/*public static final int MAX_BITS = 64;
+	public static final int MANTISSA = 52;
+	public static final int MAX_EXPONENT_BITS = 11;*/
+	public static final int MAX_BITS = 32;
+	public static final int MANTISSA = 64;
+	public static final int MAX_EXPONENT_BITS = 8;
+		public static final int BIAS = 127;
+		public static final int LONG = 64;
+		public static final int INT = 32;
+		public static final int SHORT = 16;
+		public static final int BYTE = 8;
+		//Data Type Codes...
+		public static final int STRING_TYPE = 0;
+		public static final int INT_TYPE = 1;
+		public static final int FLOAT_TYPE = 2;
+	
+	public static void main(String[] args) {
+		
+		int dataType = -1;
+		Scanner s = null;
+		boolean allGood = false;
+		try {
+			s = new Scanner( new File("testClass.txt"));
+			//System.out.println(It's open");
+			allGood = true;
+		} catch( Exception e ) {
+			System.out.println("It's not open. There is an issue opening your file. %s" + e);
+		}
+		
+		if(allGood) {
+			while(s != null && s.hasNextLine()) {
+				String nextNumIn = s.nextLine();
+				
+				if(itsNotANumber (nextNumIn)) {
+					//check for bad input
+					System.out.println(nextNumIn + " is bad input");
+					dataType = STRING_TYPE;
+				} else if(isADecimal(nextNumIn)) {
+					//check for a decimal
+					System.out.println(nextNumIn + " is a decimal");
+					dataType = FLOAT_TYPE;
+					binConvertDec(Double.parseDouble(nextNumIn));
+				} else {
+					//it's an integer
+					System.out.println(nextNumIn + " is an integer");
+					dataType = INT_TYPE;
+				}
+				
+				//now process the numbers....
+				if(dataType == INT_TYPE) {
+					String binInt = "";
+					try {
+						long x = Long.parseLong(nextNumIn);
+						//This is my conversion method...
+						binInt = binConvertInt(x);
+						System.out.println(binInt);
+						
+					} catch (Exception e) {
+						System.out.println(nextNumIn + " -- Number is too large");
+						binInt = "" + nextNumIn + " is too big for Long";
+					}
+					
+				} else if (dataType == FLOAT_TYPE) {
+					//create the 32 bit float method
+				} else {
+					//Strings??
```

Comment:
Just sysout print one line showing that there is an error. "We have an error as not int or float type!"